### PR TITLE
Pw 7598/recategorize logs

### PR DESCRIPTION
**Pull request by zenit2001**
This PR contains the recategorisation of Adyen logs in our cartridge, and a change in custom file names, so the DWEase extension can recognize the labels without extra configuration.

### CODE REVIEW

File: src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/paymentFromComponent.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ function paymentFromComponent(req, res, next) {
   });
 
   if (result.resultCode === constants.RESULTCODES.REFUSED) {
-    AdyenLogs.erorr_log(`Payment refused for order ${order.orderNo}`);
```
Comment (amihajlovski) [inline src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/paymentFromComponent.js:68]: love it 😆

File: src/cartridges/int_adyen_controllers/cartridge/controllers/Adyen.js [hunk 2]
```diff
@@ -532,7 +532,7 @@ function sessions(customer) {
 
       return responseJSON;
     } catch (error) {
-AdyenLogs.error_log(`Failed to create Adyen Checkout Session... ${error.toString()}`);
+		AdyenLogs.fatal_log(`Failed to create Adyen Checkout Session... ${error.toString()}`);
```
Comment (amihajlovski) [inline src/cartridges/int_adyen_controllers/cartridge/controllers/Adyen.js:535]: lint lint

File: src/cartridges/int_adyen_SFRA/cartridge/scripts/hooks/payment/processor/middlewares/posAuthorize.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ function posAuthorize(order, paymentInstrument, paymentProcessor) {
     terminalId,
   );
   if (result.error) {
-    AdyenLogs.error_log(`POS Authorise error, result: ${result.response}`);
+    AdyenLogs.fatal_log(`POS Authorise error, result: ${result.response}`);
```
Comment (amihajlovski) [inline src/cartridges/int_adyen_SFRA/cartridge/scripts/hooks/payment/processor/middlewares/posAuthorize.js:38]: should we stringify the response?

Comment (zenit2001) [reply src/cartridges/int_adyen_SFRA/cartridge/scripts/hooks/payment/processor/middlewares/posAuthorize.js:38]: yes we should, will add it in a bit!

Comment (zenit2001) [reply src/cartridges/int_adyen_SFRA/cartridge/scripts/hooks/payment/processor/middlewares/posAuthorize.js:38]: seems to be stringified in response

**Review by amihajlovski — APPROVED** [review]

**Review by zaiddreakh — APPROVED** [review]

**Pull request merged**